src/Components/PhotoUpload/Components/PhotoDropzone.tsx

```diff
@@ -124,22 +128,22 @@ export const PhotoDropzone: React.FC<PhotoDropzoneProps> = ({
           <Text variant="lg-display">Add photos here</Text>
         </Media>
 
-        <Text variant="sm-display" color="black60" mt={1}>
+        <Text variant={["xs", "sm-display"]} color="black60" mt={1}>
           Files supported: JPG, PNG
         </Text>
-        <Text variant="sm-display" color="black60" mt={1}>
+        <Text variant={["xs", "sm-display"]} color="black60" mt={1}>
```

Comment:
It looks like there is too much space between these two lines compared to the Figma design. Maybe we put both in one Text and add a line break between them.